packages/gatsby-transformer-cloudinary/README.md

```diff
@@ -341,134 +319,14 @@ Go to the [Gatsby Plugin Image Docs](https://www.gatsbyjs.com/docs/reference/bui
-| `originalHeight`      | `Int`       | true     | `n/a`                  | Height of the image fetched. This is required in gatsby-image to calculate the aspect ratio of the image.                                                                                                                                                                                                                                                                        |
-| `originalWidth`       | `Int`       | true     | `n/a`                  | Desired width of the image. This is required in gatsby-image to calculate the aspect ratio.                                                                                                                                                                                                                                                                                      |
-| `base64Width`         | `String`    | false    | `30`                   | Base64 width of the image.                                                                                                                                                                                                                                                                                                                                                       |
-| `version`             | `Boolean`   | false    | `false`                | Version number of image if applicable, eg. 300124291, 1241983.                                                                                                                                                                                                                                                                                                                   |
-| `ignoreDefaultBase64` | `Boolean`   | false    | false                  | If this parameter is set to true, then an API call will be made to Cloudinary when resolving your GraphQL queries to fetch a base64 image regardless of whether a `defaultBase64` image was provided for the image. This parameter can be overridden by the `alwaysUseDefaultBase64` plugin option in environments where build speed and economy of Cloudinary usage is desired. |
-
-### Arguments for `fixed`
-
-| argument | type  | default | description                                  |
-| -------- | ----- | ------- | -------------------------------------------- |
-| `height` | `Int` | `n/a`   | The height that the image should display at. |
-| `width`  | `Int` | `400`   | The width that the image should display at.  |
-
-### Arguments for `fluid`
-
-| argument   | type  | default                     | description                         |
-| ---------- | ----- | --------------------------- | ----------------------------------- |
-| `maxWidth` | `Int` | Original width of the image | The maximum width for fluid images. |
-
+<!--
+Should we not delete "aspect ratios"
```

Comment:
You should delete it

Reply:
see [fce1446](https://github.com/raae/gatsby-transformer-cloudinary/pull/8/commits/fce144627ade4927875a5bae354d08d715550172)